src/_community/rules.md

```diff
@@ -46,14 +46,38 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted in Quilt community spaces must meet the following criteria:
+
+* Projects must not be malicious - which includes (but is not limited to) malware, mods that include backdoors, or 
+  mods that harass specific people or groups of people (for example, by automatically kicking them from servers with 
+  the mod installed)
+* Projects must not be sexualised, hateful or otherwise inappropriate in nature, and this also applies to all code 
+  and project assets posted for any purpose
+* Projects must not break laws, including copyright laws - this includes piracy tools, as well as projects that 
+  ignore or otherwise break the licenses of other projects
+* Projects must not break the Terms of Service or EULA of any organisation, including (but not limited to) Mojang, 
+  Microsoft, Discord and any other community platforms we use
+* Projects must not hide their functionality, and should be upfront with what they do - however, projects with 
+  unfinished documentation or benign "Easter eggs" do not violate this rule
+* Projects must not give players an objective advantage over other players in a multiplayer setting, unless they
```

Comment:
This wording dictates what mods should be - ç'est ne comme il se pas doit. Also, the beginning and the end/subpoints list kinda diverge in meaning. I would rewrite the beginning focusing on cheats specifically rather than this unlawful deterministic catch-all

Reply:
It's intent I believe is to disallow pay to win mods

Reply:
Aside from the last sub-point (which is down there for a more sensible layout), the list of sub-points has been sorted alphabetically.

I have no idea what your review comment is supposed to mean, honestly - what's your specific criticism?

Reply:
The rule itself is about cheats. But the first sentence is too catch-all and by its wording deterministic, i.e. tries to rule the contents of mods, which is unacceptable IMO

Reply:
I can't think of anything controversial or unacceptable about regulating the content of mods that are allowed to be shared within our community spaces - perhaps you're conflating this with a general acceptable use policy for the project itself? 

We can't really dictate how people use the toolchain, nor what types of mods people write with it, but there's no reason we can't define a set of criteria that we require mods to meet - should they be promoted in our community spaces.

Reply:
My point is that you should always try to not dictate the contents of mods whenever you can. Only when contents of mods are outright malicious, or otherwise hurtful to player's machine or mental health I find wording that includes mods' contents in general acceptable

Reply:
One could make the argument that all the points presented here meet your criteria for acceptable regulation, in that case.

Reply:
I suggest this wording:
```
The following types of projects must have an opt-in mechanism, when used on servers:
```

Reply:
That does not meet the intention of the rule, where all mods providing the objective benefit require an opt-out, aside from specific categories that require an opt-**in**.

Reply:
`Objective benefit` is a catch-all. There shouldn't be rules (or laws) that leave the enforcer to decide what's objective, the rule (law) must define an enclosed list of circumstances. This is why that part is gone. And for why opt-out - consult Earthcomputer's comment.

Reply:
Earthcomputer's comment was resolved, so I don't see how it relates to your point.

Objective benefits are not left to subjective judgements. This is something that has a widely accepted definition, and there's no reason to stray from that.

Catch-alls are appropriate because exhaustively listing everything is impossible. At this point, you're bikeshedding to try to find the perfect solution - something that doesn't exist.

Reply:
> Earthcomputer's comment was resolved, so I don't see how it relates to your point.

The fact that it's resolved doesn't mean it's irrelevant

> Objective benefits are not left to subjective judgements. This is something that has a widely accepted definition

That's not true. We all know what an `insult` means, but when Russian govt used the term in the law it led to police spamming pointless "insult" lawsuits to scare people. This is a far-fetched, but eloquent example that widely accepted definitions aren't enough to make a solid rule

> you're bikeshedding to try to find the perfect solution

No, my solution is simply different from yours and you disagree with it. You can and should build exhausting list of everything. That's how good law (rule) works. You can always expand or edit the list

Reply:
> The fact that it's resolved doesn't mean it's irrelevant

The resolution both pointed out the lack of a perfect solution, and a myriad of possible approaches. You may disagree with those approaches, but that doesn't really relate to your specific point.

> That's not true. We all know what an insult means, but when Russian govt used the term in the law it led to police spamming pointless "insult" lawsuits to scare people. This is a far-fetched, but eloquent example that widely accepted definitions aren't enough to make a solid rule

I agree, this is a far-fetched example. That said, as a long-time community manager, it wouldn't be in our best interests to approach the community with bad-faith takes. We're also not necessarily a group of people with some kind of grudge against "utility"/cheat mods.

> No, my solution is simply different from yours and you disagree with it. You can and should build an exhausting list of everything. That's how good law (rule) works. You can always expand or edit the list

Exhaustive lists open up loopholes for people to try to abuse. Catch-all statements mean you can address issues as they crop up. It's a simple concept, really - the fact that cheat mod developers are now using "utility mod" to describe their projects is a perfect example of people trying to do this.

Additionally, it's unreasonable to expect us to update the policy every time the cheat-mod community comes up with a new term or a new tool. We have more important things to be doing here.

---

It's worth pointing out that this rule was worded in a way that would allow "utility"/cheat mods to exist in our community, provided those tools made good-faith attempts to cooperate with server administration teams. This is a different approach to some other communities, which ban all mods that provide any advantage, aside from a small list of categories that they like personally.

Reply:
I understand the rules update sentiment. Considering that, there's only one thing left in my point: the opt-in/opt-out detail. If my previous wording is unacceptable to moderation team, then i suggest this wording:
```
Projects must not give players an objective advantage over other players in a multiplayer setting, unless they provide mechanisms for servers to disable their features, preferably opt-in over opt-out. Such features include, but are not limited to:
- <...>
```

Reply:
I'm not personally comfortable with opt-out being the "norm" for the stuff we had listed - stating "preferably opt-in" is essentially pointless, as developers can easily just pick opt-out every time.

While it is more difficult for vanilla servers to provide a comprehensive opt-in system, requiring this at least provides some extra security - allowing everything to be opt-out instead means that vanilla servers have no recourse unless they've done the specific research they need to do to opt-out before the tools are used on the server.

Reply:
Alright, I won't mark this resolved for the time being - people are free to weigh in on this until the deadline.